tgui/packages/tgui/interfaces/CircuitAdminPanel.tsx

```diff
@@ -0,0 +1,85 @@
+import { BooleanLike } from "common/react";
+import { useBackend } from "../backend";
+import { Button, Table } from "../components";
+import { Window } from "../layouts";
+
+type CircuitAdminPanelData = {
+  circuits: {
+    ref: string;
+    name: string;
+    creator: string;
+    has_inserter: BooleanLike;
+  }[]
+}
+
+export const CircuitAdminPanel = (props, context) => {
```

Comment:
Is this a new file, or did you copy it?

Reply:
It's mine that I PR'd a week-ish ago (and got merged recently), not sure why it showed up here.